server/proxy/oauth2.go

```diff
@@ -73,6 +73,5 @@ func (p *Proxy) CompleteRemoteOAuth2(in Incoming, appID apps.AppID, urlValues ma
 
 	p.conf.Telemetry().TrackOAuthComplete(string(appID), in.ActingUserID)
 
-	p.dispatchRefreshBindingsEvent(in.ActingUserID)
```

Comment:
What happened to this function? Is it still used? If it is, is it different from the explicit new `PublishWebSocketEvent` above?

Reply:
There is not difference between `dispatchRefreshBindingsEvent` and a manual `PublishWebSocketEvent`. `dispatchRefreshBindingsEvent` is still used in the proxy. https://github.com/mattermost/mattermost-plugin-apps/blob/d3ddbe8a43b48fdcfffaa437330b072f71d8631b/server/proxy/bindings.go#L207-L212